socorro/external/postgresql/setupdb_app.py

```diff
@@ -200,7 +200,7 @@ def handle_dropdb(self, superuser_pg_url, database_name):
                 autocommit=False
         ) as db:
             if 'test' not in database_name and not self.config.force:
-                confirm = raw_input(
+                confirm = input(
```

Comment:
This changes the behavior in Python 2 which has both an `input` and a `raw_input`. We should use `six` here:

```
confirm = six.moves.input(
```

When run in Python 2, that'll use `raw_input` and when run in Python 3, that'll use `input`. That's what we want here.